core/src/main/scala/com/databricks/labs/remorph/generators/sql/ExpressionGenerator.scala

```diff
@@ -31,12 +40,18 @@ class ExpressionGenerator {
       case ir.StringType => orNull(l.string.map(doubleQuote))
       case ir.DateType =>
         l.date match {
-          case Some(date) => doubleQuote(dateFormat.format(date))
+          case Some(date) =>
+            doubleQuote(
+              LocalDate.from(ZonedDateTime.ofInstant(Instant.ofEpochMilli(date), ZoneId.of("UTC"))).format(dateFormat))
```

Comment:
Or does this change force the conversion into UTC? Perhaps we need to know if the incoming queries are assuming a default timezone other than UTC. SqlServer can do this, though it is not a good practice of course.